userspace/libsinsp/settings.h

```diff
@@ -105,3 +105,9 @@ typedef sinsp_fdinfo<int> sinsp_fdinfo_t;
 // very big JSONs returned by container inspect call
 static const unsigned MAX_JSON_SIZE_B = 500 * 1024; // 500 kiB
 
+//
+// Default metadata download settings
+//
+#define K8S_DATA_MAX_MB 100 * 1024 * 1024
```

Comment:
The constant is misnamed (otherwise this would amount to 100 TB)

Reply:
Thank you. This should be addressed in efa93793dfa05e1ae26562ec57847bf7ec700525.